src/Utils/__tests__/errors.test.ts

```diff
@@ -1,24 +1,27 @@
 import * as Sentry from "@sentry/browser"
-import { isErrorInfo, sendErrorToService } from "Utils/errors"
+import { ErrorWithMetadata, reportError } from "Utils/errors"
 
 jest.mock("@sentry/browser")
 
 describe("errors", () => {
-  describe("#isErrorInfo", () => {
-    it("returns true if an object implements the ErrorInfo interface", () => {
-      const errorInfo = { componentStack: "someString", anotherKey: 0 }
-      expect(isErrorInfo(errorInfo)).toBe(true)
+  describe("#reportError", () => {
+    const scope = { setExtra: jest.fn() }
+    const err = new Error("some error")
+    const errWithMetadata = new ErrorWithMetadata("some message", {
+      foo: "bar",
     })
-    it("returns false if an object does not implement the ErrorInfo interface", () => {
-      const notErrorInfo = { anotherKey: 0 }
-      expect(isErrorInfo(notErrorInfo)).toBe(false)
+    it("does not call setExtra on scope if the error has no metadata", () => {
```

Comment:
Can you add newlines between blocks?

Reply:
I'm not sure if this is related, but lately I've noticed sporadic automatic white-space removal. Not sure if its related to prettier precommit hook or tslint autofix (or something else entirely) but whatever it is is def new. Not sure how to reproduce.

Reply:
This time it was me 😄